mir-ci/mir_ci/robot_resources/screencopy.resource

```diff
@@ -0,0 +1,11 @@
+*** Keywords ***
+Get Center
+    [Arguments]    ${region}
+    ${point}=    Evaluate
+    ...    {"x": int((${region}[left] + ${region}[right]) / 2), "y": int((${region}[top] + ${region}[bottom]) / 2)}
+    RETURN    ${point}
+
+Add Displacement
```

Comment:
How about `Displace`? Or do we want to avoid single-word keywords to avoid polluting the namespace?

Reply:
` Displace` looks good. There's no such keyword in the built-in library.